adafruit_is31fl3731/matrix.py

```diff
@@ -48,7 +56,7 @@ def pixel_addr(x, y):
     # for animation. Buffering the full matrix for a quick write is not a
     # memory concern here, as by definition this method is used with PIL
     # images; we're not running on a RAM-constrained microcontroller.
-    def image(self, img, blink=None, frame=None):
+    def image(self, img: str = None, blink: bool = False, frame: int = 0):
```

Comment:
`img` here should be Image that comes from an import like from PIL import Image I believe.

`frame` should keep it's old default of `None` I think and become `Optional[int]` type